docs/sandbox/transferanalyzer.md

```diff
@@ -0,0 +1,14 @@
+# Direct transfer analyzer module
+
+- Gard Mellemstrand, Entur, Norway
+
+## Changelog
+
+### May 20th 2019
+
+- Added the direct transfer analyzer module
+
+### Documentation
+
```

Comment:
Add 1 or 2 sentences to describe the functionality; higher level description of what this does.